charts/backup/values.yaml

```diff
@@ -13,6 +13,13 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+sftpConfig:
+  # -- # Mounted as ConfigMap to /etc/ssh/ssh_known_hosts
+  knownHosts: []
+  privateKeys:
+    # -- # secret mounted to /root/.ssh/
+    fromSecret: ~
```

Comment:
Can you rename this to `existingSecret`? This has turned out as a kind of standard over time.